### ISSUE TITLE
Holding an Item causes grey text

### ISSUE THREAD

**Issue by l33tdaniel** [alphawolf918/Zollern-Galaxy#88]
For whatever reason, when I hold an item, all text in Minecraft turns grey. This includes grey text in the F3 tab, chat, and the text that shows which item is being held. This is my first issue I've ever made - please let me know what is needed from me if anything

**Comment by alphawolf918** [comment]
Yeah, that's been an annoying thing for a while. No idea what causes it. It's on my Trello board, though. Had it up there before, but took it down because it seemed to resolve itself. Seems not, though. Sorry I can't be more help. Wish I knew what caused it.

**Comment by ROMVoid95** [comment]
@l33tdaniel Is it any item in the game? Or a particular item? And does it go back to normal after you stop holding said item?

**Comment by l33tdaniel** [comment]
@ROMVoid95 This happens for any item in the game, and it immediately goes back to normal when I switch back to an open hand

**Comment by alphawolf918** [comment]
Just to clarify: when you say "any item in the game," does that mean any game in Minecraft and its mods, or any item just from Zollern Galaxy?

**Comment by l33tdaniel** [comment]
This happens with both vanilla minecraft items, other modded items, and items from Zollern galaxy specifically. If Zollern Galaxy is removed, the problem doesn't persist. In fact, having only Zollern Galaxy installed still causes the issue (I specifically tested this to narrow it down so that we could ensure Zollern Galaxy was causing the issue). If we remove all mods and go back to forge with no mods, I don't have the issue at all. 

I've found that an easy way to test this is to hold open F3 since it displays more text on the screen thus making it more noticeable when it goes to that darker shade of grey instead of being white. I run a small Minecraft server for my friends, and they all have the same issue as well.

Thank you for your continued help

**Comment by alphawolf918** [comment]
Thank you for the in depth description, that is very helpful.

We are still looking into this issue, but as of yet have been unable to find the cause. It's been sent to an expert who can hopefully shed some light on this. Until then, I'll leave this issue up, and hopefully we can get this sorted out somehow.

**Comment by dhgameryt** [comment]
its really annoying
